### ISSUE TITLE
Система координат

### ISSUE THREAD

**Issue by DocItSage** [MAIResearchGroup/Paraglider#11]
Думаю стоит пересмотреть модель в очередной раз. Начать стоит с выбора системы координат, в которых будут проходить расчеты. Ваши предположения?

**Comment by DocItSage** [comment]
Судя по всему все согласны с тем, что используется стартовая СК?

**Comment by ElMigele** [comment]
Да, надо использовать СК связанную с начальным положением. На данный момент это удобнее всего. Если конечно правильно составить графики направления сил и не запутаться с углами
